library.lp

```diff
@@ -0,0 +1,10 @@
+(librepcb_library 0120d71f-d2f7-4e52-8a11-80ada981ad5f
+ (name "Adafruit")
+ (description "Adafruit breakout boards. Inofficial")
```

Comment:
For consistency with other libs:

```suggestion
 (description "Adafruit breakout boards (inofficial).")
```

Reply:
Hmm. I noticed it is less consistent in the repository names. But that is not really related.